schedule/yast/maintenance/register_image.yaml

```diff
@@ -0,0 +1,9 @@
+name: register_image_for_migration_mu_jobs
+description:    >
+  Boot an image and register it for migration MU jobs.
+schedule:
+  - installation/bootloader_start
+  - boot/boot_to_desktop
+  - migration/online_migration/register_system
```

Comment:
have you considered to use command line tool to do the registration, would be more stable or even faster perhaps?

Reply:
Yes, I tried to registered the system by SUSEConnect. But on SLES12SP5, when registering WE by SUSEConnect command line, it has an error about the third party repo of nvidia which would make the testing fail, so I switched to yast2 scc.

Reply:
what does yast2 scc that doesn't give us that error? we should do the same.

Reply:
I use SUSEConnect to register now. The error is about nvidia repo's metadata, it has already worked fine if we retry and refresh the repos.